src/lib/language.js

```diff
@@ -0,0 +1,68 @@
+import store from '../store';
+
+class Language {
```

Comment:
You don't need all of this stuff. The idea is to export the method(s) you want to call.
Please fix it.